build_helper.sh

```diff
@@ -58,9 +58,6 @@ function prepare_upload_data {
   rm -rf ${LOCAL_PATH}
   mkdir -p ${LOCAL_PATH}
   mv handler.zip ${LOCAL_PATH}/lambda.zip
-}
-
-function prepare_cloudformation_code {
```

Comment:
OK, well then better merge it.

The Makefile can be discussed later I believe.
I do have a preference for the Makefile over scripting, as it seems more logical to use it when dealing with compiled languages, as the project/collaborator grows, you might not want to have to do full compilations all the time for a minor edit; which is something a script isn't able to do easily.
The "scripting part" (condition/loop/output management/etc) is also something that can be done via the Makefile, so I don't really feel the need to write a script for that.